pkg/hwinfo/hardware.go

```diff
@@ -263,7 +263,10 @@ const (
 // HardwareDevice represents a hardware component detected in the system.
 type HardwareDevice struct {
 	// Class represents the type of the hardware component.
-	Class HardwareClass `json:"-"`
+	Class HardwareClass `json:"class,omitempty"`
```

Comment:
After scrolling through the report, this actually doesn't add much and seems redundant.
The class_list is fine though.

```suggestion
	Class HardwareClass `json:"-"`
```

Reply:
The class is implied by the section the device is placed in, and if it's unknown we look to the first entry in the class list, so I'm happy removing it from the json output